Sources/LambdaConverters.Deployment/LambdaConverters.nuspec

```diff
@@ -1,15 +1,15 @@
 ﻿<?xml version="1.0" encoding="utf-8" ?>
 <package>
     <metadata>
-        <id>LambdaConverters</id>
+        <id>LambdaConvertersAndCo</id>
```

Comment:
Guessing you didn't mean to include the changes to this file.

Reply:
I would not change the `id` to avoid package upgrade issues.

Reply:
Yes, I didn't mean to include the recent two commits to the Pull Request. I made a new nuget package under my profile, because I didn't get any answers to the pull request and I needed to consume the changes as nuget package (https://www.nuget.org/packages/LambdaConvertersAndCo/). This is my first pull request on github, so I need to figure out how to remove the last two commits from the pull request.

Reply:
I figured it out. I shouldn't have pushed changes semantically distinct to the pull request on the master branch as long as the pull request is open. Thus, the last commit on master reverts the two commits before it. Now the pull request should be as I originaly intended it. I am sorry for the inconvenience.